src/components/createConnector.js

```diff
@@ -62,12 +68,27 @@ export default function createConnector(React) {
       return { slice };
     }
 
+    bindActionCreators(dispatch) {
+      if (!this.props.actionCreators) {
+        return {};
```

Comment:
I think a sane default is the current behavior: injecting just the `dispatch` prop. Let's remove this early exit, and instead specify `dispatch => ({ dispatch })` as the default prop value for `actionCreators`.